### PR TITLE
Make noarch: python (with variants)

### LINKED ISSUE #44: make noarch: python with pywin32-on-windows
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

We could now consider using https://github.com/conda-forge/pywin32-on-windows-feedstock (and `run_constrained`) to get our pin without doing n x m builds. I am a bit torn, as we _have_ occasionally found cross-platform bugs for upstream... but I am also very lazy.

### Installed packages

```shell
pywin32>
```

### Environment info

```shell
windows?
```

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

References:
- alternative to #95

closes #49 
closes #95 
closes #44

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,36 +9,54 @@ source:
...
+{% endif %}
 
-{% set skip = ["test_not_on_path", "test_path_priority"] %}
-# linux shebang lines have a length limit longer than the placeholder test prefix
-{% set skip = skip + ["test_argv0"] %}  # [linux]
+# combine for -k
 {% set skip = '-k "not (' + (skip | join(' or ')) + ')"' %}
 
-# coverage is very slow on pypy
+{% set cov = "" %}
```
Comment (minrk) [inline recipe/meta.yaml:56]: I think we can do without the coverage tests. Enforcing test coverage for packaging tools doesn't make a lot of sense to me.

**Comment by minrk** [comment]
Thanks, I think this works. I added a tweak to the constraints, so the cpython build requires the `cpython` metapackage, rather than excluding pypy.

File: recipe/meta.yaml (hunk 1 above)
Comment (bollwyvl) [reply recipe/meta.yaml:53]: While I'm _slightly_ more clued into the stately pace `jupyter_core` dev than some of the other things I maintain, I've found that sudden, rapid drops in coverage are the best way to find e.g. new optional dependencies. I've been generally trying to keep them on important packages, where possible.

**Pull request merged**